bzt/cli.py

```diff
@@ -64,7 +64,6 @@ def __init__(self, options):
         self.exit_code = 0
 
     @staticmethod
-    @run_once
```

Comment:
THis was preventing shared use from multiplying logging. For example, in Taurus-server.